operator/v1/types_console.go

```diff
@@ -149,6 +149,10 @@ type DeveloperConsoleCatalogCustomization struct {
 	// +kubebuilder:validation:Optional
 	// +optional
 	Categories []DeveloperConsoleCatalogCategory `json:"categories,omitempty"`
+	// disable types which are shown in the developer catalog.
+	// +kubebuilder:validation:Optional
+	// +optional
+	DisableTypes []string `json:"disableTypes,omitempty"`
```

Comment:
The first word references the YAML/JSON type anywhere else. Recommend to align the doc to Quickstarts and add actions. See https://github.com/openshift/api/blob/874dd9fe45490eb7347db7422582e43ddffa2e3d/operator/v1/types_console.go#L192-L196 and https://github.com/openshift/api/blob/874dd9fe45490eb7347db7422582e43ddffa2e3d/operator/v1/types_console.go#L200-L206

```suggestion
	// disableTypes is a list of developer catalog IDs that are not shown to users.
	// +kubebuilder:validation:Optional
	// +kubebuilder:validation:MinItems=1
	// +optional
	DisableTypes []string `json:"disableTypes,omitempty"`
```